src/locales/ar/messages.po

```diff
@@ -11,31 +11,31 @@ msgstr ""
 "PO-Revision-Date: \n"
 "Last-Translator: \n"
 "Language-Team: \n"
-"Plural-Forms: \n"
+"Plural-Forms: nplurals=6; plural=(n==0 ? 0 : n==1 ? 1 : n==2 ? 2 : n%100>=3 && n%100<=10 ? 3 : n%100>=11 && n%100<=99 ? 4 : 5);\n"
 
 #: src/ModulesList.js:175
 msgid "(unidentified)"
 msgstr "(غير معرّف)"
 
 #: src/Resource.js:144
 #~ msgid "All Items"
-#~ msgstr "جميع العناصر"
+#~ msgstr ""
```

Comment:
is this ok?

Reply:
I believe so, yes.  This was automatically generated, and you'll notice that the entire section is commented out anyways.  I did a project search for "All Items" and it looks like it's not being used anymore.

Reply:
oh ok, a bit weird that we removed that translation